Assets/UI/TabWidgets/Buttons/MainButton/MainButtons/ZoomButton.gd

```diff
@@ -0,0 +1,22 @@
+extends WidgetButton
+class_name ZoomButton
+
+enum ZOOM_DIRECTION {IN, OUT}
+@export var zoom_direction: ZOOM_DIRECTION
+
+func _ready() -> void:
+	pressed.connect(_on_ZoomButton_pressed)
+
+	if zoom_direction == ZOOM_DIRECTION.IN:
```

Comment:
You can use math here, please read _on_TabWidget_button_rotate_left_pressed()

You can also use composition to avoid this case